DSP_CopyInserters/CopyInserters.cs

```diff
@@ -5,12 +5,24 @@
 using BepInEx;
 using HarmonyLib;
 using UnityEngine;
+using UnityEngine.UI;
 
 namespace DSP_Mods.CopyInserters
 {
     [BepInPlugin("org.fezeral.plugins.copyinserters", "Copy Inserters Plug-In", "1.1.0.0")]
     class CopyInserters : BaseUnityPlugin
     {
+        public static bool copyEnabled = true;
+        public static List<UIKeyTipNode> allTips;
+        public static UIKeyTipNode tip;
+        void Update()
+        {
+  
+            if (Input.GetKeyUp(KeyCode.LeftControl))
```

Comment:
```suggestion
            if (Input.GetKeyUp(KeyCode.LeftControl) && IsCopyAvailable())
```
I think it's a bit confusing to have the copy mode turn off if I use control to transfer some items or something in between copies.

Reply:
Agreed. Using TAB instead as it should never overlaps